### DISCUSSION TITLE
快速收藏功能在哪里生效

### DISCUSSION THREAD

**Discussion by overthink13** [the1812/Bilibili-Evolved#3514] [提问 / Q&A] [answered]
快速收藏怎么只有在稍后再看里打开的视频页面才能出现啊，在主页里打开一个视频没有快速收藏功能

**Comment by Geeyun-JY3** [comment, marked as answer]
再具体些？给个截图？（别开新版视频界面：#2615）

**Reply by overthink13** [reply]
谢谢，切换到旧版视频界面解决了。
不过在新版视频界面中，如果放到稍后再看里，再在稍后再看里打开视频，就会出现快速收藏功能。具体我也不太清楚。
![image](https://user-images.githubusercontent.com/88617084/180365326-8878dead-2fe7-4079-a34f-3ba15c7e7c06.png)
![image](https://user-images.githubusercontent.com/88617084/180365497-38b6518f-1f67-4364-bceb-35b327e3f5bf.png)

**Reply by Geeyun-JY3** [reply]
新版视频界面没适配，就不用管啦
